orttraining/orttraining/python/training/onnxblock/blocks.py

```diff
@@ -47,7 +47,10 @@ def __call__(self, *args, **kwargs):
 
         output = self.build(*args, **kwargs)
 
-        onnx.checker.check_model(self.base, True)
+        if accessor._GLOBAL_ACCESSOR.has_path:
+            onnx.checker.check_model(accessor._GLOBAL_ACCESSOR.path, True)
```

Comment:
Does the checker fail if the ModelProto is > 2GB and that's why we need to pass a path and let it load another copy using that?